tests/ecosystem/deployment/test_deployment.py

```diff
@@ -23,6 +23,11 @@ def test_deployment(pvc_factory, pod_factory):
                 'ocs_registry_image'
             )
             ocs_install_verification(ocs_registry_image=ocs_registry_image)
+
+            nb_eps = config.DEPLOYMENT.get('noobaa_endpoints')
+            if nb_eps > 1:
```

Comment:
Hey @ebenahar ,

Sorry to be late here, but I don't think it should be part of this test_deployment test as it's mainly should care about deployment verification and should not do any extra changes in the deployment.

If this change should be part of our deployment this could should be called from deploy_ocs function I think:
https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/deployment/deployment.py#L439

Reply:
Hi @petr-balogh thanks for the input.
I consider putting this change in the deploy_ocs function function but the problem with that is that changes will be required in https://github.com/red-hat-storage/ocs-ci/blob/ea0417e5aad2d4c27328b34f23f830c06d55a9f4/ocs_ci/ocs/resources/storage_cluster.py#L39 as we are expecting a fixed number of Noobaa resources and if we change the max of Noobaa endpoints before this verification without somehow adjusting it, the verification will fail. 

I will try to think how to solve this issue and send a PR

Reply:
@ebenahar I see you are patching this:
https://github.com/red-hat-storage/ocs-ci/pull/2314/files#diff-9829dc4eedf17bed5719295e70c7fa84R512
``` params = f'{{"spec":{{"endpoints":{{"maxCount":{nb_eps},"minCount":1}}}}}}'```

So by default if possible - no value should be changed and we should have default behavior, only if specific config value is provided, than you can compare the value with what is in the config?

Otherwise I guess you can read this value dynamically from that object instead of for verification to read it from config?
From value: `maxCount`. IIUC ?

WDYT?

Reply:
In PR https://github.com/red-hat-storage/ocs-ci/pull/2366 I changed the default to be max of 2 endpoints.

Out of the box, `maxCount` is 1. this condition of checking the value from the config is in order to change the NooBaa CR.

Reply:
Reported https://github.com/red-hat-storage/ocs-ci/issues/2376 to track the change needed here